data/Alexxanderr20.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Edward Munoz" 
+  "bio": "Aspiring software engineer who is focused on a front-end tech stack, but is open to any kind of     software stack to learn and grow." ,
+  "links" : [
+   {
+    "name": "Follow me on LinkedIn" ,
+    "url": "https://www.linkedin.com/in/edward-alex-munoz/",
+    "icon": "FaLinkedIn"
```

Comment:
Please change your icon name to `FaLinkedin`.
docs:  https://linkfree.io/icons

Reply:
```suggestion
    "icon": "FaLinkedin"
```